index.html

```diff
@@ -1,579 +1,998 @@
-<a>decentralized identifiers</a> created in this way prevent namespace conflicts
-and semantic ambiguity.
+          Fragments identify sub resources of a resource identified with a DID
+          URL.
         </p>
 
         <p>
-An extensibility model this dynamic does increase the implementation burden.
-Software written for such a system needs to determine if accepting
-<a>DID documents</a> with extensions is acceptable based on the risk profile of
-the application. Some applications might choose to accept but ignore extensions,
-others might choose to only accept certain extensions, while highly secure
-environments might disallow extensions. These decisions are up to the
-application developers and are specifically not the domain of this
-specification.
+          The interpretation of the fragment is determined by the mime type.
         </p>
+      </section>
+    </section>
 
-        <p>
-Implementations MUST produce an error when an extension JSON-LD Context
-overrides the expanded URL for a term specified in this specification. To avoid
-the possibility of accidentally overriding terms, developers SHOULD scope their
-extensions. For example, the following extension scopes the new
-<code>PhotoStreamService</code> term so that it can only be used within the
-<code>service</code> property:
-        </p>
+    <section class="informative">
+      <h3>Representations</h3>
 
-        <pre class="example nohighlight" title="Scoping terms in a JSON-LD Context">
-{
-    "@context": {
-        <span class="highlight">"service": {
-            "@id": "https://w3id.org/did#service",
-            "@context": {
-                "PhotoStreamService": "https://example.com/vocab#PhotoStreamService"
-            }
-        }</span>
-    }
-}
-        </pre>
+      <p class="note">
+        See [[DID-CORE]] for normative requirements associated with developing
+        <a href="https://www.w3.org/TR/did-core/#representations">Representations</a>.
+      </p>
+
+      <p>
+        The [[DID-CORE]] specification describes representations and the
+        [[DID-SPEC-REGISTRIES]] supports the registration of an unbounded
+        number of alternative representations.
+      </p>
+
+      <p>Each representation has unique pros and cons.</p>
+
+      <p>
+        DID Methods are responsible for producing a DID Document in the
+        requested representation.
+      </p>
+
+      <p>
+        DID Method implementers may choose to return verification methods in
+        alternate formats for representations, for example, an implementer
+        might prefer to return verification material encoded with
+        <code>publicKeyJwk</code> for <code>application/did+json</code> and
+        <code>publicKeyBase58</code> for <code>application/did+ld+json</code>.
+      </p>
+
+      <p>
+        If a DID Method supports both <code>application/did+json</code> and
+        <code>application/did+ld+json</code> if it recommended that they both
```

Comment:
```suggestion
        <code>application/did+ld+json</code>, it is recommended that they both
```
but then there's the comment by @peacekeeper